### PR TITLE
Adds an Arnold Palmer to the game with a new sprite for it.

### PR DESCRIPTION
**Pull request by Probe1**
Anon in the thread asked for it, and here we are.  The tasty mix of lemonade and iced tea.
Requires 1 part Iced Tea, 1 part Lemonade.  Results in 

```
    adj_temp = -5
    adj_sleepy = -3
    adj_dizzy = -1
    adj_drowsy = -3
```

Which in normie is it makes you cool off, wakes you up, and makes you a little less dizzy.  I tested it and it works, but the sprite leaves something to be desired.

image

### CODE REVIEW

**Comment by Shadowmech88** [comment]
For the sprite, try to make two separate layers, where each color progressively fades to more transparent, and then overlap them.
Either that or use dithering.

**Comment by Probe1** [comment]
@Shadowmech88 Better? 
image

**Comment by Shadowmech88** [comment]
Much, but I'd still thin out the overlap area by one pixel. In your example pic, they blend where they intersect, but the blending doesn't go much past that.

**Comment by Probe1** [comment]
image

arnoldpalmer_edited-1

**Comment by ihadtoregisterforthis** [comment]
Add to wiki pls

**Comment by Probe1** [comment]
image

File: html/changelogs/Probe1.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+ author: Probe1
```
Comment (Probe1) [inline html/changelogs/Probe1.yml:1]: >a space before author
I FUCKED IT UP.

**Pull request merged**